gfx/UltimateCataclysm/pngs_normal_32x32/furniture/f_freezer/freezer.json

```diff
@@ -0,0 +1,6 @@
+[
+{
+  "id": [ "freezer", "f_freezer", "ap_freezer" ],
```

Comment:
the item looking exactly the same as the furniture and the appliance is going to be confusing. At least the item should be different

Reply:
I made the fix and will commit it ( Them ) soon